sympy/solvers/solvers.py

```diff
@@ -586,50 +530,49 @@ def solve(f, *symbols, **flags):
         >>> solve(x**4 - 1, x, set=True)
         ([x], {(-1,), (1,), (-I,), (I,)})
 
-    * single expression with no symbol that is in the expression
+    Single expression with no *symbol* that is in the expression:
```

Comment:
If the parameter name is `symbols` (plural) then we should only use asterisks where we refer precisely to that parameter name. So this "symbol" should be in normal text. There are other examples of this below

Reply:
Good catch! Making those changes now.